packages/openapi3-parser/lib/parser/oas/parseSchemaObject.js

```diff
@@ -108,10 +108,14 @@ const typeToElementNameMap = {
  * Normalises result into an ArrayElement of StringElement
  */
 function parseType(context) {
+  const { namespace } = context;
+
   let types;
   let isValidType;
 
-  if (context.isOpenAPIVersionMoreThanOrEqual(3, 1)) {
+  const isOpenAPI31OrHigher = R.always(context.isOpenAPIVersionMoreThanOrEqual(3, 1));
```

Comment:
Perhaps a suggestion for another PR

A `context.isOpenAPI31OrHigher` could be added as that is something we are using the `isOpenAPIVersionMoreThanOrEqual` almost always for IIRC. Then it could be destructured nicely in
```js
const { namespace, isOpenAPI31OrHigher } = context;
```